assets/src/models/createDetourMachine.ts

```diff
@@ -12,6 +19,11 @@ import {
 import { DetourShape, FinishedDetour } from "./detour"
 import { fullStoryEvent } from "../helpers/fullStory"
 import { type, optional, coerce, date, string } from "superstruct"
+import { detourApiActor } from "../state/detourApiActor"
+
+const tracked = () => ({
+  saved: false,
+})
```

Comment:
I think I'd like a different word here than `tracked`? Maybe `autosave`? Open for debate